packages/runtimes/js/src/loaders/browser/preload-loader.js

```diff
@@ -0,0 +1,14 @@
+const cacheLoader = require('../../cacheLoader');
+
+module.exports = cacheLoader(function preloadJSBundle(bundle, priority) {
+  var link = document.createElement('link');
+  link.charset = 'utf-8';
+  link.rel = 'preload';
```

Comment:
Do we want to use "modulepreload" for ES module output format? I think that will let the browser parse and compile the code in advance in addition to just loading it. Only supported by chromium browsers though... https://developers.google.com/web/updates/2017/12/modulepreload